packages/stark-build/config/webpack.common.js

```diff
@@ -166,14 +166,16 @@ module.exports = options => {
 				// see https://github.com/angular/angular-cli/issues/8594
 				{
 					test: /(?:\.ngfactory\.js|\.ngstyle\.js|\.ts)$/,
-					use: METADATA.AOT ? [buildOptimizerLoader, "@ngtools/webpack"] : ["@ngtools/webpack"]
+					use: METADATA.AOT ? [buildOptimizerLoader, "@ngtools/webpack"] : ["@ngtools/webpack"],
+					exclude: METADATA.AOT ? [helpers.root("node_modules/prettier")] : []
```

Comment:
Actually, this exclude is not needed to make the build working. Could you please remove it ? 😊

Reply:
It's gone

Reply:
It's gone